access/ms-teams/validate.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	cards "github.com/DanielTitkov/go-adaptive-cards"
+	"github.com/google/uuid"
+	"github.com/gravitational/teleport-plugins/lib"
+	"github.com/gravitational/teleport-plugins/lib/plugindata"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/trace"
+)
+
+// validate installs the application for a user if required and sends the Hello, world! message
+func validate(configPath, userID string) error {
+	var uid = userID
+
+	lib.PrintVersion(appName, Version, Gitref)
+	fmt.Println()
+
+	c, err := LoadConfig(configPath)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	fmt.Printf(" - Checking application %v status...\n", c.MSAPI.TeamsAppID)
+
+	b, err := NewBot(c.MSAPI, "local", "")
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	teamApp, err := b.GetTeamsApp(context.Background())
+	if trace.IsNotFound(err) {
+		fmt.Printf("Application %v not found in the org app store. Please, ensure that you have the application uploaded and installed for your team.", c.MSAPI.TeamsAppID)
+		return nil
```

Comment:
I think we should return an error here :thinking: 
Otherwise it's _validated_

Reply:
We do not want an error output and backtrace in this case. validate.go represents the console command which sends a fake message, it returns only fatal errors (networking, etc). Otherwise, it nicely shows the result to a user.

Reply:
Got it :+1: 

I would expect the `validate` method to return an error and we would then print the message on the caller
It's fine this way